packages/curvenote-cli/src/submissions/submit.ts

```diff
@@ -71,15 +72,15 @@ export async function submit(session: ISession, venue: string, opts?: SubmitOpts
     path: gitInfo?.path,
     commit: gitInfo?.commit,
   };
-
-  session.log.info(`📍 Submitting using key: ${chalk.bold(key)}`);
+  if (opts?.draft) key = undefined;
```

Comment:
@stevejpurves - Once we have a better idea of workflows we can probably delete this (could even delete now...). Things are fixed so even without this line, draft submission workflow is ok.

The difference is: with this line, drafts create new "inaccessible" work with no key and a new submission version; without this line, drafts only create work _versions_ on the existing keyed work (still a new submission version).